app/src/main/res/values/strings.xml

```diff
@@ -704,24 +704,24 @@
-    <string name="feed_load_error">Error loading feed</string>
+    <string name="feed_load_error">Could not load feed</string>
     <string name="feed_load_error_account_info">Could not load feed for \'%s\'.</string>
-    <string name="feed_load_error_terminated">The author\'s account has been terminated.\nNewPipe will not be able to load this feed in the future.\nDo you want to unsubscribe from this channel?</string>
+    <string name="feed_load_error_terminated">The author\'s account has been terminated.\nNewPipe will not be able to load this feed in the future.\nUnsubscribe from this channel?</string>
     <string name="feed_load_error_fast_unknown">The fast feed mode does not provide more info on this.</string>
     <string name="feed_use_dedicated_fetch_method_title">Fetch from dedicated feed when available</string>
     <string name="feed_use_dedicated_fetch_method_summary">Available in some services, it is usually much faster but may return a limited amount of items and often incomplete information (e.g. no duration, item type, no live status)</string>
-    <string name="feed_use_dedicated_fetch_method_enable_button">Enable fast mode</string>
-    <string name="feed_use_dedicated_fetch_method_disable_button">Disable fast mode</string>
-    <string name="feed_use_dedicated_fetch_method_help_text">Do you think feed loading is too slow? If so, try enabling fast loading (you can change it in settings or by pressing the button below).\n\nNewPipe offers two feed loading strategies:\n• Fetching the whole subscription channel, which is slow but complete.\n• Using a dedicated service endpoint, which is fast but usually not complete.\n\nThe difference between the two is that the fast one usually lacks some information, like the item\'s duration or type (can\'t distinguish between live videos and normal ones) and it may return less items.\n\nYouTube is an example of a service that offers this fast method with its RSS feed.\n\nSo the choice boils down to what you prefer: speed or precise information.</string>
+    <string name="feed_use_dedicated_fetch_method_enable_button">Fast mode</string>
+    <string name="feed_use_dedicated_fetch_method_disable_button">No fast mode</string>
+    <string name="feed_use_dedicated_fetch_method_help_text">Loading too slow? Try turning on fast loading (you can change it in settings or by pressing the button below).\n\nNewPipe offers two feed loading strategies:\n• Fetching the whole subscription channel, which is slow but complete.\n• Using a dedicated service endpoint, which is fast but usually not complete.\n\nThe difference between the two is that the fast one usually lacks some information, like the item\'s duration or type (can\'t distinguish between live videos and normal ones) and it may return less items.\n\nYouTube is an example of a service that offers this fast method with its RSS feed.\n\nSo the choice boils down to what you prefer: speed or precise information.</string>
```

Comment:
What is "Fast loading" called in the settings?

Reply:
```suggestion
    <string name="feed_use_dedicated_fetch_method_help_text">Loading too slow? Try turning on \'Fast loading\' (in the settings, or by pressing the button below).\nThis skips downloading the entire channel, but may be imprecise.</string>
```

Reply:
Maybe "Fast loading" is better. But do not commit that big change, the old description was more descriptive

Reply:
Don't make it "fast loading"? The old way was to call it "fast loading" here, and then "fast mode" for the setting.

The old description here does what better?
I just want a 
problem described and 
the action relating to it. Then I can
Find out where to do it 
gauge whether there is some loss involved.

What the original does is:
Problem described
Action relating to it
That the action can be undone
Where the action can be undone
Reference the name of the app
Enumerate how many feed loading strategies there are
Description of method 1.
Technical details of method 1
Description of method 2
Technical details of method 2
Difference between the two stated
Technical details of why there is a difference
Something potential that can happen with one option
Explanation of just one service that is covered by method 2
New juxtaposition between the two methods

If a conclusion is needed for a description help text, the mark is missed.